components/brave_wallet/common/zcash_utils_unittest.cc

```diff
@@ -48,4 +48,13 @@ TEST(ZCashUtilsUnitTest, PubkeyToTransparentAddress) {
                 false));
 }
 
+TEST(ZCashUtilsUnitTest, ExtractTransparentPart) {
+  auto transparent_part = ExtractTransparentPart(
+      "u14nl3xm252a24u4tr9ml453ptz4uv4lam2ytau94nmnf9sf3cx5vdrh9ksf3f24d7jk8l35"
```

Comment:
nit: is this a standardized test vector or just copied from somewhere else?

Reply:
There are a lot of conditions in `ExtractTransparentPart` that should have tests like https://github.com/brave/brave-core/pull/21460/files#diff-be755d19a8d4855f9e7b39a9a4ff8424bce9e7daf73546f7710f416e7fe349ceR177  and https://github.com/brave/brave-core/pull/21460/files#diff-be755d19a8d4855f9e7b39a9a4ff8424bce9e7daf73546f7710f416e7fe349ceR182 etc...
A single test with an expected correct result is not sufficient and that applies to other unit tests here as well

Reply:
I've added other testcases and also linked to librustzcash test vectors when available